modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -105,14 +105,16 @@ public class ObjectRelationshipLocalServiceImpl
 	@Indexable(type = IndexableType.REINDEX)
 	@Override
 	public ObjectRelationship addObjectRelationship(
-			long userId, long objectDefinitionId1, long objectDefinitionId2,
```

Comment:
I believe you only added this parameter in the ADD methods. I believe we have to add this parameter and validate it in the update methods.

Reply:
I will check with Gabriel/Camacho if we will allow the edition of the ERC in the Object Relationship. But, I believe that we have to allow to maintain the pattern.